assets/in

```diff
@@ -27,6 +27,7 @@ configure_git_ssl_verification $payload
 
 uri=$(jq -r '.source.uri // ""' < $payload)
 branch=$(jq -r '.source.branch // ""' < $payload)
+tag_filter=$(jq -r '.source.tag_filter // ""' < $payload)
```

Comment:
I don't think `tag_filter` should be documented as part of the get, as it is documented already in the [source documentation](https://github.com/concourse/git-resource/#source-configuration). 

Also, this is not a new option, it has been added in #31. In this PR I check if it has been defined in the source.

But it makes sense to update the documentation to mention that `in/get` will report back the tag as reference.